pkg/cloud/services/compute/instance.go

```diff
@@ -217,11 +157,21 @@ func (s *Service) createInstance(eventObject runtime.Object, clusterName string,
 		}
 
 		if err := s.deletePorts(eventObject, portList); err != nil {
-			s.logger.V(4).Error(err, "failed to clean up ports after failure", "cluster", clusterName, "machine", instanceSpec.Name)
+			s.logger.V(4).Error(err, "failed to clean up ports after failure")
```

Comment:
```suggestion
			s.logger.V(4).Error(err, "Failed to clean up ports after failure")
```

Reply:
Of course this is a log message at Error, not an actual error, so ST1005 wouldn't apply. Might as well fix it as we're already touching the line.

That said, I am hoping to entirely get rid of this 😁